<ui version="4.0" >
 <class>AddNewInvDialog</class>
 <widget class="QDialog" name="AddNewInvDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add new folder</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="dirLabel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dirEdit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dirButton" >
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="invLabel" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>New folder name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="invEdit" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox" >
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Header</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="headerEdit" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Comments</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="commentsEdit" />
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Subdirectories max depth</string>
       </property>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QSpinBox" name="maxDepthEdit" >
          <property name="suffix" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="newInvBox" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Create new folder
for each subdirectory</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>21</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="authorEdit" >
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="minimumSize" >
        <size>
         <width>110</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Coloured</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="colouredCBox" >
       <property name="currentIndex" >
        <number>-1</number>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="forUsBox" >
       <property name="text" >
        <string>Only for us</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="minimumSize" >
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="text" >
          <string>Common description
 (will appear on 
 all the photos) </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="descriptionEdit" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QTextEdit" name="headerEdit_2" >
       <property name="html" >
        <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">NOTES: &lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; font-weight:600;">"Max depth" &lt;span style=" font-weight:400;">field is used to set the maximum depth of subdirectories from which images will be loaded:&lt;/span>&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">* If 0&lt;/span>, none of subdirectories will be looked for images.&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">* If &amp;gt;1&lt;/span>, you can use the checkbox to specify whether PhotoManager places the images from subdirectories into the same folder or creates a new folder for each subdirectory;&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;span style=" font-weight:600;">* If 1 and checkbox not set&lt;/span>,  all images from the first level subdirectories will be placed into the same folder, giving their "Sub-folder" property the name of sub-directory. &lt;/p>&lt;/body>&lt;/html></string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
